content/docs/best-practices/start-your-dataform-project.md

```diff
@@ -110,10 +110,20 @@ config { type: "table" }
 select * from ${ref("charges")}
 ```
 
-## Use schedules to run transformations at a regular interval
+## Use tags to manage your schedules
 
-Schedules update the datasets defined by your project in your data warehouse at a specified frequency
+Tags can be added to SQLX config blocks to create collection of actions and datasets. When starting your project, it can useful to create to start using at least the following two tags:
```

Comment:
```suggestion
Tags can be added to SQLX config blocks to create collection of actions and datasets. When starting your project, it can useful to start using the following two tags:
```